tstest/integration/vms/vms_test.go

```diff
@@ -415,20 +446,39 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 		for port := range ipMap {
 			port := port
 			t.Run(port, func(t *testing.T) {
-				config := &ssh.ClientConfig{
-					User:            "ts",
-					Auth:            []ssh.AuthMethod{ssh.PublicKeys(signer), ssh.Password("hunter2")},
-					HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+				t.Parallel()
+				time.Sleep(5 * time.Second)
+
+				hostport := fmt.Sprintf("127.0.0.1:%s", port)
+
+				const maxRetries = 5
```

Comment:
This retry loop helps to make things a bit faster, centos sometimes is slow at starting its sshd. I don't know why they don't use socket activation.

Reply:
Add a comment then. A comment in the PR only will get lost for future readers.

Reply:
Done